src/cart/utils/memcheck-cart.supp

```diff
@@ -81,6 +81,82 @@
    fun:crt_hg_ctx_init
    ...
 }
+{
+   FI leak 6
+   Memcheck:Leak
+   match-leak-kinds: possible
+   ...
+   obj:/usr/lib64/librdmacm.so.1.1.22.1
+   ...
+}
+{
+   FI leak 7
+   Memcheck:Leak
+   match-leak-kinds: indirect
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 8
+   Memcheck:Leak
+   match-leak-kinds: possible
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 9
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   FI leak 10
+   Memcheck:Leak
+   match-leak-kinds: definite
+   ...
+   fun:fi_ini
+   fun:fi_getinfo@@FABRIC_1.3
+   ...
+}
+{
+   spdk leak 0
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:calloc
+   fun:rte_log_init
```

Comment:
Is there a rte_log_fini() function we should call?

At least one other person has also found this, so we should report upstream if there is no way to resolve it with the current spdk. https://tracker.ceph.com/issues/36033

Reply:
The `rte_log_init()` funcion is called using a constructor here:
[eal_common_log.c](https://github.com/spdk/dpdk/blob/3fcb1dd395a870c64450f151a53338034b832906/lib/librte_eal/common/eal_common_log.c#L356)

I couldn’t find a `rte_log_fini()` or similar function. Can I file a ticket to investigate/address it?

Reply:
I think it's low priority, but we do need to check for these functions, and it would be good to file a upstream ticket.  This is low priority however, but carrying these suppressions does give us a maintenance burden.

Reply:
See below comment for full details. I am wondering if it is bug that we should report to the SPDK team.

Valgrind documentation says:
"still reachable" means your program is probably ok -- it didn't free some memory it could have.

Reply:
"still reachable" is one of the types of memory leak we check for, currently there are zero still reachable regions in the results of our testing, and the threshold for failure is also zero, although we do suppress some.  If you're re-spinning this PR then can you add a comment that we've checked for a rte_log_fini() function and there isn't one, and it'll save time when it comes to reviewing the contents of the suppression file in future.